### PR TITLE
Add shadydom@1.0.8 w/ git auto-update

### LINKED ISSUE #10243: [Request] Add shadydom
**Library name:** shadydom
**Git repository url:** https://github.com/webcomponents/shadydom
**npm package url(optional):** 
**License(s):**
**Official homepage:**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #10243, cc @sorvell, @azakus

Pull request for issue: #10243
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/webcomponents/shadydom
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/@webcomponents/shadydom
 * License and its reference: BSD-3-Clause
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 22
   - Count of stars: 81
   - Count of forks: 40
 * NPM download stats (optional):
   - Downloads in the last day:  33
   - Downloads in the last week: 164
   - Downloads in the last month: 607

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sashberd** [comment]
@sufuf3 The PR message was updated. FYR

**Comment by sufuf3** [comment]
Helped rebase to the latest master.

File: ajax/libs/shadydom/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "webcomponents",
+    "polyfill",
+    "shim"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webcomponents/shadydom.git"
+  },
+  "author": "The Polymer Authors",
+  "homepage": "https://webcomponents.org/polyfills",
```
Comment (extend1994) [inline ajax/libs/shadydom/package.json]: Though this follows the upstream, and "Shadow DOM polyfill" is introduced in the link, the content of this page seems for another GitHub repo https://github.com/WebComponents/webcomponentsjs.

Comment (sashberd) [reply ajax/libs/shadydom/package.json]: @extend1994 I did not understand, is there need to change something?

Comment (extend1994) [reply ajax/libs/shadydom/package.json]: @sashberd No for now. But I didn't sure about it. That's why I request review from Peter.

**Pull request merged**